.github/workflows/mvn-verify-check.yml

```diff
@@ -30,7 +30,50 @@ env:
     -Dmaven.scaladoc.skip
 
 jobs:
+  cache-dependencies:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: ['8']
```

Comment:
it may not be easy or possible to get an output of a matrix job per https://github.com/actions/runner/pull/2477#issuecomment-1501003600 and we do not really need a matrix job here for now 
 ```suggestion
```